rules/framework.bzl

```
@@ -363,6 +366,13 @@ def _get_framework_files(ctx, deps):
     virtualize_frameworks = feature_names.virtualize_frameworks in ctx.features
     if not virtualize_frameworks:
         framework_manifest = ctx.actions.declare_file(framework_dir + ".manifest")
+        if ctx.attr.link_dynamic != True:
+            infoplist_in = _merge_root_infoplists(ctx)
+            infoplist_out = infoplist_in
+            infoplist_out = [paths.join(
+                framework_dir,
+                "Info.plist",
+            )]
```

I wish we didn't have to bring `ctx.attr.link_dynamic` in here but it seems necessary to maintain the functionality from before. Little simplification of this code though before merge, please:
```suggestion
        if not ctx.attr.link_dynamic:
            infoplist_in = _merge_root_infoplists(ctx)
            infoplist_out = [paths.join(
                framework_dir,
                "Info.plist",
            )]
```